src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -2922,73 +2777,15 @@
                 "datasource": {
                     "uid": "${prometheusds}"
                 },
-                "definition": "label_values({__name__=~\"pg_up|mysql_up|mongodb_up|proxysql_mysql_status_active_transactions\"}, replication_set)",
```

Comment:
same here

Reply:
Duplicated definition of `replication_set` which is anyway removed as it doesn't make sense to keep it in this dashboard